### PR TITLE
chore: add Mockito and Mockk libraries for unit and UI testing

### PR DESCRIPTION
**Pull request by JitterKiller**
- Added Mockito core, inline, and Android libraries for unit testing.
- Added Mockk core, agent, and Android libraries for unit and UI testing.
- Updated `libs.versions.toml` with versions for Mockito and Mockk.
- Updated `build.gradle.kts` to include Mockito and Mockk dependencies for testing.

### CODE REVIEW

**Review by Sebbayra — DISMISSED** [review]
This correctly imports mockito for the testing, works well.

File: app/src/main/java/com/arygm/quickfix/MainActivity.kt
Comment (AnotherMedo) [inline app/src/main/java/com/arygm/quickfix/MainActivity.kt:1]: Drop these changes are they are not related to the PR

File: app/src/main/java/com/arygm/quickfix/ui/Activity.kt
Comment (AnotherMedo) [inline app/src/main/java/com/arygm/quickfix/ui/Activity.kt:1]: Same here

File: app/src/main/java/com/arygm/quickfix/ui/Announcement.kt
Comment (AnotherMedo) [inline app/src/main/java/com/arygm/quickfix/ui/Announcement.kt:1]: Same here

File: app/src/main/java/com/arygm/quickfix/ui/Calendar.kt
Comment (AnotherMedo) [inline app/src/main/java/com/arygm/quickfix/ui/Calendar.kt:1]: Same here

File: app/src/main/java/com/arygm/quickfix/ui/Map.kt
Comment (AnotherMedo) [inline app/src/main/java/com/arygm/quickfix/ui/Map.kt:1]: Same here

File: app/src/main/java/com/arygm/quickfix/ui/Other.kt
Comment (AnotherMedo) [inline app/src/main/java/com/arygm/quickfix/ui/Other.kt:1]: Same here

**Review by AnotherMedo — CHANGES_REQUESTED** [review]
Drop the changes made in `src/main/java/con/arygm/quickfix` as they are not related to the addition of Mockito and Mockk. You might have created your branch inside another working branch 😅

**Review by AnotherMedo — APPROVED** [review]
This is great! thank you for being so reactive. I will fix the formatting asap

**Comment by AnotherMedo** [comment]
Exceptionally merging this as it is needed for PRs that fix pass the CI
